docs/source/fitfunctions/FunctionQDepends.rst

```diff
@@ -0,0 +1,30 @@
+.. _func-FunctionQDepends:
+
+================
+FunctionQDepends
+================
+
+.. index:: FunctionQDepends
+
+Description
+-----------
+
+This fitting function is the base class for all fitting functions that have:
+
+- A range of energy transfers as their one-dimensional domain from which to take values
+- The magnitude of the momemtum transfer, :math:`Q`, as one of their attributes.
+
+Fitting functions for QENS data depending on :math:`Q` should derive from this class.
+
+There are two ways to update attribute :math:`Q` in the fit funtion:
+
+- The user inputs a particular value.
+- The spectrum contains :math:`Q` in addition to the range of energy transfers.
+
+Conflict may arise when both options are available. In that case, priority is given to the :math:`Q`-value contained
```

Comment:
Is there a particular reason why the workspace `Q` takes precedence? This way it's not possible to override it if for some reason `Q` in the workspace is incorrect.